File: 2026/07-July/23-Thursday/solution.cs
public class Solution
{
    public int UniqueXorTriplets(int[] nums)
    {
        int n = nums.Length;

        if (n <= 2)
            return n;

        int bits = 0;
        int value = n;

        while (value > 0)
        {
            bits++;
            value >>= 1;
        }

        return 1 << bits;
    }
}

File: 2026/07-July/23-Thursday/solution.c
int uniqueXorTriplets(int* nums, int numsSize) {
    if (numsSize <= 2)
        return numsSize;

    int bits = 0;
    int temp = numsSize;

    while (temp > 0) {
        bits++;
        temp >>= 1;
    }

    return 1 << bits;
}
